pkg/apis/serving/v1beta1/route_types.go

```diff
@@ -89,10 +89,17 @@ type TrafficTarget struct {
 	// +optional
 	LatestRevision *bool `json:"latestRevision,omitempty"`
 
-	// Percent specifies percent of the traffic to this Revision or Configuration.
-	// This defaults to zero if unspecified.
+	// Percent indicates that percentage based routing should be used and
+	// the value indicates the percent of traffic that is be routed to this
+	// Revision or Configuration. `0` (zero) mean no traffic, `100` means all
+	// traffic.
+	// If "percent" appears on more than one Traffic Target then the sum of
```

Comment:
What if it is on 1 target, but it's not 100?

Reply:
that's an error. Just checked and even if `percent` is just one one of them it must be `100`.

Reply:
well the text does not convey that (at least the way I read it).

Reply:
ah good point - I missed what you were getting at. See if what I just pushed is any clearer.

Reply:
Yep, this is better.